### ISSUE TITLE
pfe-tab: Focus tab order

### ISSUE THREAD

**Issue by coreyvickery** [patternfly/patternfly-elements#807]
Update the tab order to focus each tab in the component from left to right. Currently, only the Active tab is focused (see home page for example).

**Comment by kylebuch8** [comment]
@coreyvickery - I'm pretty sure this is the expected behavior. Look at the Keyboard Support for the Tab Key in the WAI Authoring Practices example for tabs.

https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html#kbd_label

**Comment by coreyvickery** [comment]
@kylebuch8 Hey Kyle, thanks for posting that. As it exists now, I don't think the component on the home page (and elsewhere perhaps) exhibits that behavior, which should be fixed.

**Comment by coreyvickery** [comment]
Opening a new issue for all changes to pfe-tabs. Closing.
